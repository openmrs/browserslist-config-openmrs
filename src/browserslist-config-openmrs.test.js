const browserslist = require("browserslist");
const queries = require("./browserslist-config-openmrs.js");

describe("browserslist-config-openmrs", () => {
  it(`exports a valid browserslist config that is what we want for OpenMRS`, () => {
    const browsers = browserslist(queries);
    expect(browsers).toMatchSnapshot();
  });
});
