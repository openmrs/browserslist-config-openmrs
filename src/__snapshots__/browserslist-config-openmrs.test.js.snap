// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`browserslist-config-openmrs exports a valid browserslist config that is what we want for OpenMRS 1`] = `
Array [
  "chrome 81",
  "chrome 80",
  "chrome 79",
  "chrome 78",
  "chrome 77",
  "edge 80",
  "edge 79",
  "edge 18",
  "firefox 75",
  "firefox 74",
  "firefox 73",
  "opera 66",
  "opera 65",
  "opera 64",
  "safari 13",
  "safari 12.1",
  "safari 12",
]
`;
